### ISSUE TITLE
Integrate Land Manager and Scout homepages with Default Arches User Profiles

### ISSUE THREAD

**Issue by mradamcox** [legiongis/hmsf-md#147]
Since we first built this HMS app, Arches has added a user profile page that looks like this (/user):

![image](https://user-images.githubusercontent.com/10427268/82093956-c10e0f00-96c1-11ea-9395-710768d2bbe9.png)

We should migrate the Land Manager and Scout home pages into a customized version of this default Arches view, instead of leaving them out and completely separate as they are now. The look and feel of the pages should be more like the current FPAN pages than the Arches default, but the plumbing should be based on Arches patterns, and the content should include the new content shown above: Notification Settings, Arches Collector Projects, etc.

Unsure at this point whether that means we'll need to build from base-manager or not... Before approaching this ticket we'll meet to discuss design and content more thoroughly.

**Comment by coherit** [comment]
I'm on a roll with the design work. @mradamcox How would you like to handle the scout profile form? My inclination is to work it in to the "my account" panel in the user template because there's some redundant form fields. By default, the my account panel is in a display state -- the user has to click the "Edit" button to get into form state -- this would change the UX for fpan scouts. I've not mucked around with the scout profile before, so let me know what you think. 

![image](https://user-images.githubusercontent.com/3217017/130874719-dff04e3f-d7a4-4b80-8513-7780cdb84eee.png)

**Comment by mradamcox** [comment]
Looking great. Don't worry about the profile section for now, I should get the proper form passed to the view before we start dealing with that, also it will be a different form based on whether it's an admin, scout, or land manager so that'll be it's own bit of work. It's possible we'll need separate templates for each of those sections that are conditionally included.

**Comment by mradamcox** [comment]
Ok, having looked further into how the user profile form is managed in this template, it's not as straightforward as I was expecting. 

The actual Django form is not passed to the template at all, only the user.first_name, last_name, etc. variables. When the Edit button is clicked, a few input elements are displayed that are pre-populated with these variables. When these inputs are posted with Save, then the data is sent to the post() method of the view where the `ArchesUserProfileForm` is finally instantiated with that data, validated, and then saved. I think what I would like (which is how we have the original Scout profile interface set up) is for the actual ScoutProfileForm to be passed to the original view and then edited directly in the template.

So for this ticket, let's ignore the profile form altogether and just focus on the rest of the design, like the site access rules summary and content. Once all of that is taken care of, we can merge it in for demoing, and after that address the profile form content itself.

**Comment by coherit** [comment]
This is ready for review. Let me know what you would like tweaked -- tried to make it as close to the original hms and landmanager home pages as possible. https://github.com/legiongis/fpan/tree/147_profile_design

**Comment by mradamcox** [comment]
Thanks. It's looking good but I do have a few things more in mind.

First, and probably the easiest, I'd like to get the site access section more readable. I could generate a string from the profile models that could give a verbose description, but could you first try using that dictionary of permissions rules to generate something like:

> Your access to archaeological sites is based on ___ (agency, management area, etc.). You can view any sites managed by  _____ (agency name, area name(s))

Then, below in the accessible sites section, let's start with a `<ul>` that lists site names as links to their /report/ (each site followed by indented links to its Scout Reports), and then we'll go from there. We could add "Create New Scout Report" link next to each site name as well or something like that.

Second, I think we should rework the top bar and header of this page. Conceptually we could really collapse the original header with the new one you have added, but technically I'm not sure how well that would work. I'm also not crazy about the way that the blue or green background pattern fits with the blue or green sidebar, but this is what we get when we mix the fpan-base with base-manager, which is essentially what we're doing. Let's think on it for now...

Thank you!

**Comment by coherit** [comment]
Sounds good -- as for the navigation bars -- along the top I could try restyling the white navigation bar to the maroon color (so it's more like the original "home" page). I could also style the side navigation bar as the user's accent color -- e.g. orange for scout and navy for landmanager, which should be more of a contrast with the background rice paper. I didn't tackle admin yet...

**Comment by mradamcox** [comment]
Yeah, I was thinking of that for the side bar, but it may be a little jarring when the user goes to the search page, for example, and the sidebar goes from orange to blue. Perhaps the side and top can be maroon on the profile page (which would be like the home page) until the user goes to search/add resource. That may still be jarring, but a little less at least.

**Comment by coherit** [comment]
There appears to be something wrong with the way the accessible_sites dictionary is building its scout report list. E.g. SJxxxxx - Matanzas Site 2 has two scout reports linked to its resource report: 1c7a5055-5dd7-45d3-863b-725f5c0f8b33 and 0346d229-27cc-40f7-b8bb-e57090c065b2. But, in the accessible_sites dictionary, the scout_reports values are: ['1a2d1325-8a10-4380-832e-621eeebc63c5', '1a2d1325-8a10-4380-832e-621eeebc63c5'], which happens to be the Mantanzas Site 2 resource id (but listed twice, once for each scout report presumably). So, perhaps the model is returning the wrong resource instance id?

![image](https://user-images.githubusercontent.com/3217017/132578114-cbabbf86-dd29-4385-b43b-bf5560a9522c.png)

**Comment by mradamcox** [comment]
Indeed, I think you're exactly right. I'll push a fix to that branch as
soon as I can.

**Comment by coherit** [comment]
This branch now has the style edits we discussed above. The sidebar nav is maroon, and the upper nav is maroon and mimics the old hms/home pages as much as possible. Also removed the extra banner. 

![image](https://user-images.githubusercontent.com/3217017/132903799-f158be7c-84e8-463a-9424-47496810dd6b.png)

![image](https://user-images.githubusercontent.com/3217017/132903879-1d03a208-d16d-49c1-baa6-0e8f823c7b5e.png)

![image](https://user-images.githubusercontent.com/3217017/132903945-e0d04976-f71e-42b4-8dbf-ff620273a2af.png)

**Comment by mradamcox** [comment]
@coherit  thanks again for all this. I went through it yesterday, made some updates, and then deployed. All set! I will leave the actual integration with the scout profile form for another ticket, I spent a while fiddling with it yesterday, but not enough to get it fully functional. Closing this ticket now.
